Add behavioral guidelines in line with summary

eloquence opened the pull request:
This draws liberally from the cited codes (Tor, OpenStack, Django)
as well as FPF's organizational code (which is also based on Django's).

eloquence committed:
Add behavioral guidelines in line with summary

This draws liberally from the cited codes (Tor, OpenStack, Django)
as well as FPF's organizational code (which is also based on Django's).

dachary commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+Behavioral Guidelines
+================================================================================
+
+As a member of the community, we ask you to:
```
I would skip this because the CoC could apply in contexts where _community_ has to be defined and the meaning of _we_ is also subject to interpretation. For instance when this CoC is used by FPF as a non-profit, the **we** is different from when this CoC is used by groups of people who have no ties with FPF.

eloquence replied:
I think that using "we" in some contexts has a positive, necessary, affirming quality: *We* own our commitment to these principles. *We* take it seriously. *We* are accountable for enforcing the code.  The passive voice in contexts like this is evasive.

But I understand what you're saying about it being important to identify who is described by the word. Perhaps language like "We, your fellow community members, .." could work, to state clearly that in this context, we are speaking as peers independent of our organizational affiliations.

dachary commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  motives.
+
+- Be welcoming, considerate, and respectful. We strive to be a community that
+  welcomes and supports people of all backgrounds and identities. This includes,
```
What about **People of all backgrounds and identities deserve to be welcomed and supported**

dachary commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  family status, political belief, religion and mental and physical ability.
+
+- Be careful in the words you choose. We are a community of professionals, and
+  we conduct ourselves professionally. Exclusionary behavior is absolutely
```
I suggest we skip **We are a community of professionals, and we conduct ourselves professionally.** so this CoC can also be used by people who are not professionals. Or it could be replaced by a similar sentence that does not sugest this only concerns people who are professionally working on a project.

dachary commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
+following behaviors:
```
For the same reasons as above (avoiding we and community to not tie the CoC to a specific context), we could write instead **You are expected to never engage in the following behaviors:**

dachary commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  will consider a violation in spirit (e.g., clearly behaving in a damaging or
+  obnoxious manner in a way not explicitly specified) to be no different from
+  any other violation of this code. That includes trolling and other forms of
+  consistently disruptive behavior.
```
I would suggest rewording this to not use _we_. Something along the lines of **Certain behavior represents a violation of this code in spirit (e.g., clearly behaving in a damaging or obnoxious manner in a way not explicitly specified; trolling and other forms of consistently disruptive behavior) and are no different from any other violation of this code.**

ghost commented:
Comments inline, thanks for getting this moving :-)

heartsucker commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
```
Can we add misgendering and deadnaming to this?

heartsucker replied:
Can we also add something about addiction and drug/substance use or lack thereof? i know a lot of people don't like when communities are like Boyz Drinkin' Clubz or whatever. Those who use/abstain shouldn't be given a hard time about it.

heartsucker commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  consistently disruptive behavior.
+
 ================================================================================
 License and Attribution
```
Just since I'm trying to keep up with licensing / attribution, if you took any of this directly or nearly, could you make sure to include that down here. I'd like to give credit where it's due. :)

eloquence replied:
Thanks for flagging, added FPF and OpenStack as inspirations, as well.

heartsucker reviewed (commented):
Thanks for keeping this moving. I've been juggling a lot and I'm glad you were able to pick up where I left off :)

A couple of minor notes I'd like to add too.

heartsucker commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying
+    information ("doxing")
+
```
I think this would be a good spot to mention general boundary violations. I know that's kinda vague, but if people need to not be talked to or not be asked any personal questions or not share food or like whatever, there may be a reason. This kinda covers one of the major cases of the "obey the spirit, not just the letter" and might make it easier to enforce the grey areas should be need to. For example.

> Part of having a diverse community withe diverse backgrounds, people you interact with will have different levels of comfort with different topics or social behaviors. Please respect people's stated boundaries, and do not attempt change these boundaries through your actions. This includes repeated requests or demands against a stated boundary.

I think there could be a better way to phrase that, but I'm not exactly an expert on this. :neutral_face:

eloquence replied:
I really appreciate that point, and I think "Respect people's stated personal boundaries" is pretty clear. Unpacking that further risks going more into territory where people want to rules-lawyer about it; I think we can keep it short and sweet.

eloquence committed:
Remove language around "professionals"

As noted in comments, that can be seen as exclusionary, esp. if the
word is taken literally

eloquence committed:
Frame guidelines as expectations per comments

eloquence committed:
Add "Respect people's stated personal boundaries" per comment

eloquence committed:
Add reference to FPF's code of conduct as one of the inspirations

eloquence commented:
Thanks for the review and comments! :) I've addressed most of them and would appreciate another look.

Regarding the use of the word "we", I don't agree fully with @dachary's comments. I think it's important for us to (as a community) "own" this process, so I do think in various locations we can and should say *we*. As for the concern that it's not clear who's speaking, I would suggest adding the following language to the first occurrence of the word "We" in the preamble:

"As your fellow members of this community, we ..."

That makes it clear that it is a "We" that is encompassing us as individuals, regardless of our organizational affiliations. With that qualifier, I think the use of the word "we" in other parts of the CoC is not problematic, and in fact affirming our recognition of our shared responsibilities. Thoughts?

eloquence committed:
Add reference to OpenStack CoC as inspiration

heartsucker commented on CoC.txt:
```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+As a member of the SecureDrop community, you are expected to never engage in the
+following behaviors:
+
+- Intimidation, harassments, or insults.  This includes but is not limited to:
```
Could we get deadnaming/misgendering here? I feel like that one is harmful and easily dismissed enough by "well I didn't know" that it should be called out.

eloquence replied:
Sorry, missed that on first pass. Done!

eloquence committed:
Add misgendering, deadnaming
